evalml/pipelines/components/estimators/regressors/exponential_smoothing_regressor.py

```diff
@@ -0,0 +1,137 @@
+"""Holt-Winters Exponential Smoothing Forecaster."""
+import numpy as np
+import pandas as pd
+from skopt.space import Integer
+
+from evalml.model_family import ModelFamily
+from evalml.pipelines.components.estimators import Estimator
+from evalml.problem_types import ProblemTypes
+from evalml.utils import import_or_raise, infer_feature_types
+
+
+class ExponentialSmoothingRegressor(Estimator):
+    """Holt-Winters Exponential Smoothing Forecaster.
+
+    Currently ExponentialSmoothingRegressor isn't supported via conda install. It's recommended that it be installed via PyPI.
+
+    Args:
```

Comment:
Can you add all of the new input parameters to the docstring

Reply:
Great catch 😅